docs/architecture.md

```diff
@@ -43,4 +43,4 @@ Gatling Runner Pod runs multiple containers. The diagram below shows how those c
 - gatling-result-transferer
   - Uploads the simulation.log file to to user-selected Cloud Storage (can be configured in `.spec.cloudStorageSpec`)
 
-📝 It is noted that gatling-waiter and gatling-runner run as init containers and gatling-result-transferer as a main container in Gatling Runner Pod in the case of generating an aggregated Gatling result report. However it's also noted that gatling-waitner runs as an init container and gatling-runner run as a main container in the case of not generating the report.
+📝 It is noted that **from gatling-operator-v0.8.1** gatling-waiter runs as an init container and both gatling-runner and gatling-result-transferer runs as main containers in a Gatling Runner Pod. However, **before gatling-operator-v0.8.1** both gatling-waiter and gatling-runner run as init containers and gatling-result-transferer as a main container in the case of generating an aggregated Gatling result report while gatling-runner runs as a main container in the case of not generating the report.
```

Comment:
v0.8.1 is going to be the next version ( which of course hasn't been released yet at this point)